src/app_ui/address.rs

```diff
@@ -55,11 +55,22 @@ pub fn ui_display_pk_base58(public_key: &crypto::PublicKeyBe) -> Result<bool, Ap
         const NEAR_LOGO: NbglGlyph =
             NbglGlyph::from_include(include_gif!("icons/app_near_64px.gif", NBGL));
 
-        let mut review: NbglAddressReview = NbglAddressReview::new()
+        let review: NbglAddressReview = NbglAddressReview::new()
             .glyph(&NEAR_LOGO)
             .verify_str("Confirm Public Key");
 
-        Ok(review.show(out_buf.as_str()))
+        let res = review.show(out_buf.as_str());
+        let status = NbglReviewStatus::new();
+        match res {
+            true => {
+                status.status_type(StatusType::Address).show(true);
+            }
+            false => {
+                status.status_type(StatusType::Address).show(false);
+            }
+        }
```

Comment:
Wouldn't it be easier to just do:

```suggestion
        NbglReviewStatus::new().status_type(StatusType::Address).show(res);
```

?

I see that this code is duplicated in a number of other places.

Reply:
Yes indeed. Will update the code accordingly, thank you !